packages/client/mutations/handlers/handleEditTask.ts

```diff
@@ -15,7 +15,7 @@ const handleEditTask = (payload: RecordProxy, store: RecordSourceSelectorProxy)
   if (isEditing) {
     // handle multiple socket connections
     for (let ii = 0; ii < taskEditors.length; ii++) {
```

Comment:
nit: `forEach`

Reply:
It calls `return` in the loop, so looks like I can't simply replace it with `forEach` here.